exec/java-exec/src/main/codegen/templates/ParquetOutputRecordWriter.java

```diff
@@ -268,10 +268,112 @@ public void writeField() throws IOException {
     consumer.endField(fieldName, fieldId);
   </#if>
     }
+
+    <#if mode.prefix == "Repeated">
+    /**
+     * Writing list field according to logical type rules
```

Comment:
```suggestion
     * Writelist field according to logical type rules
```

Reply:
I'll drop the javadoc, intent is explained in parent one.